pkg/backingstore/backingstore.go

```diff
@@ -427,7 +429,28 @@ func RunDelete(cmd *cobra.Command, args []string) {
 	backStore.Namespace = options.Namespace
 	backStore.Spec = nbv1.BackingStoreSpec{}
 
-	if !util.KubeDelete(backStore) {
+	nbClient := system.GetNBClient()
+
+	continueDelete := true
+	poolinfo, err := nbClient.ReadPoolAPI(nb.ReadPoolParams{Name: backStore.Name})
+	if err != nil {
+		if rpcErr, isRPCErr := err.(*nb.RPCError); !isRPCErr || rpcErr.RPCCode != "NO_SUCH_POOL" {
```

Comment:
I think this statement is too long for a one liner...
You can break it:
```golang
	rpcErr, isRPCErr := err.(*nb.RPCError)
	if !isRPCErr || rpcErr.RPCCode != "NO_SUCH_POOL" {
```

Reply:
fixed